model/CMakeLists.txt

```
@@ -409,12 +409,33 @@ foreach (xlen IN ITEMS 32 64)
                         --coq-lib mem_metadata
                         --coq-output-dir ${CMAKE_CURRENT_BINARY_DIR}
                         # The prefix of the output files.
-                        -o "riscv_${arch}"
+                        -o "coq_${arch}"
                         # Input files.
                         ${sail_srcs}
                 )
                 add_custom_target(generated_coq_${arch} DEPENDS "coq_${arch}.?")
             endif()
+
+            # Build Lean definitions
+            if (variant STREQUAL "lean")
+                string(TOUPPER ${arch} arch_uppercase) 
+                add_custom_command(
+                    DEPENDS ${sail_srcs}
+                    VERBATIM
+                    OUTPUT "lean_${arch_uppercase}.lean"
```

I think this needs to be `Lean`:

```suggestion
                    OUTPUT "Lean_${arch_uppercase}.lean"
```

---

Ok fair enough!